### PR TITLE
Update Australis yml for cn4 to dai2 replacement

### PR DESCRIPTION
**Pull request by nbwhite**
Signed-off-by: Nathan B White <nathan.b.white@intel.com>

### CODE REVIEW

File: configurations/aurora/scripts/Australis.yml [hunk 1]
```diff
@@ -373,6 +373,7 @@ dai-networking-topics:
       avro: true
   events1: # Plain Text topics only
       topics:
+        - SYSLOG
```
Comment (paul-amonson) [inline configurations/aurora/scripts/Australis.yml:376]: Why 2 SYSLOG topics?

**Review by paul-amonson — CHANGES_REQUESTED** [review]
Where is node removal from a CN rack?

File: configurations/aurora/scripts/Australis.yml (hunk 1 above)
Comment (nbwhite) [reply configurations/aurora/scripts/Australis.yml:376]: Whoops, must have misclicked something while editing.

**Review by paul-amonson — APPROVED** [review]
I would recommend testing before merge. ;)

**Comment by nbwhite** [comment]
> I would recommend testing before merge. ;)
Yep, already done.

**Review by avincigu — APPROVED** [review]

**Pull request merged**